src/libs/CheckPDFDocument/index.ts

```diff
@@ -0,0 +1,19 @@
+import pdfWorkerSource from 'pdfjs-dist/legacy/build/pdf.worker';
+import {pdfjs} from 'react-pdf';
+import type CheckPDFDocument from './types';
+
+if (!pdfjs.GlobalWorkerOptions.workerSrc) {
+    pdfjs.GlobalWorkerOptions.workerSrc = URL.createObjectURL(new Blob([pdfWorkerSource], {type: 'text/javascript'}));
+}
```

Comment:
What's the purpose of this?

Reply:
its needed for `pdfjs` to work, reference here https://mozilla.github.io/pdf.js/examples/